2gp-demo/force-app/main/default/classes/LicensingUtil.cls

```diff
@@ -53,14 +36,9 @@ public with sharing class LicensingUtil {
             return '';
         }
     }
-    
-    @TestVisible
-    private static Boolean isPackagingOrg() {
-        return String.isNotBlank(getNamespace()) && packageNamespaces.contains(getNamespace()) && String.isBlank(getPackageId());
-    }
         
     public static boolean userHasPackageLicense() {
```

Comment:
add a comment above "returns true if package licensing for current org is set to "site-wide". Scratch orgs and sandboxes are always on site-wide licensing for packages"